### DISCUSSION TITLE
Improvise See Removed Comment

### DISCUSSION THREAD

**Discussion by jcperil** [Docile-Alligator/Infinity-For-Reddit#1273] [Not planned Ideas] [answered]
If you click `See Removed Comment`, redact username of commenter and only show content of the removed comment. This respects the privacy of the commenter.

**Comment by MysteriousOrb2000** [comment, marked as answer]
Not to be rude here, but why would you want to gimp a downstream scrapper tool? Besides, I use it this a lot and love to see who made the comments. If the author, wants privacy on a deleted content, (s)he can changes accounts.

**Reply by Wladefant** [reply]
Agree
